src/core/QCheck.ml

```diff
@@ -86,15 +86,20 @@ let _opt_sum a b = match a, b with
 
 let sum_int = List.fold_left (+) 0
 
-(* Included for backwards compatibility, pre 4.13 *)
-let string_fold_right f s acc =
-  let len = String.length s in
+let _fold_righ length get f x acc =
```

Comment:
```suggestion
let _fold_rigth length get f x acc =
```

Reply:
eehh, I believe it is spelled `right`.
Now allow me to include misspell myslef - casting the first stone and all... :wink:

Reply:
Thanks for spotting this typo!